src/main/scala/mesosphere/mesos/TaskBuilder.scala

```diff
@@ -182,12 +182,16 @@ class TaskBuilder(
         // TODO(portMappings)
         // TODO(nfnt): Other containers might also support port mappings in the future.
         // If that is the case, a more general way than the one below needs to be implemented.
-        val containerWithPortMappings = c match {
-          case docker: Container.Docker => docker.copy(portMappings = boundPortMappings)
+        val containerWithPortMappingsAndTaskIdLabel = c match {
```

Comment:
nit: perhaps instead of appending the names of all the fields we've changed, just call this `updatedContainer`?